src/ripple/app/paths/PathRequest.cpp

```diff
@@ -585,11 +585,13 @@ PathRequest::findPaths (std::shared_ptr<RippleLineCache> const& cache,
         {
             Json::Value jvEntry (Json::objectValue);
             rc.actualAmountIn.setIssuer (sourceAccount);
-            jvEntry[jss::source_amount] = rc.actualAmountIn.getJson (JsonOptions::none);
+            jvEntry[jss::source_amount] =
+              rc.actualAmountIn.getJson (JsonOptions::none);
```

Comment:
Sorry to be picky, but indents should be 4 spaces. Soon, we'll have everything clang-formatted and this will get easier for everyone...

Reply:
@thejohnfreeman updated!